ServiceFinder/ServiceFinder.DAL/Repositories/AssistanceCategoryRepository.cs

```diff
@@ -23,6 +23,18 @@ public override async Task<List<AssistanceCategoryEntity>> GetAllAsync(Cancellat
                 .Include(ac => ac.Assistances)
                 .ToListAsync(cancellationToken);
         }
+        public async override Task<AssistanceCategoryEntity> UpdateAsync(AssistanceCategoryEntity entity, CancellationToken cancellationToken)
+        {
+            var existingEntity = await Query.AsNoTracking()
+                .FirstOrDefaultAsync(e => e.Id == entity.Id, cancellationToken);
+
+
+            entity.CreatedAt = existingEntity?.CreatedAt ?? entity.CreatedAt;
+
```

Comment:
Don't get why you need this line if you have interceptor

Reply:
forgot(